samples/powerpoint/text/get-set-textrange.yaml

```
@@ -183,46 +183,82 @@ script:
     language: typescript
 template:
     content: |-
-        <section class="ms-Fabric ms-font-m">
+        <section class="description">
             <p>This sample shows how to get selected text, and how to select specific text.</p>
         </section>
-        <section class="ms-Fabric samples ms-font-m">
+        <section class="samples">
             <h3>Try it out</h3>
             <p>Before choosing the <b>Get selected text range</b> button, add text to a slide and select some text.</p>
-            <button id="getSelectedTextRange" class="ms-Button"><span class="ms-Button-label">Get selected text range</span></button>
-            <button id="setSelectedTextRange" class="ms-Button"><span class="ms-Button-label">Selects first 10 characters of selected shape</span></button>
-            <button id="changeColor" class="ms-Button"><span class="ms-Button-label">Change color of selected text</span></button>
-            <button id="saveTextSelection" class="ms-Button"><span class="ms-Button-label">Save text selection</span></button>
-            <button id="loadTextSelection" class="ms-Button"><span class="ms-Button-label">Load the saved selection</span></button>
-            <button id="getSelectedTextRangeProperties" class="ms-Button"><span class="ms-Button-label">Get selected text range properties</span></button>
+            <button id="getSelectedTextRange">Get selected text range</button>
+            <button id="setSelectedTextRange">Selects first 10 characters of selected shape</button>
```

Unrelated to your change, but could you update this? Thx

```suggestion
            <button id="setSelectedTextRange">Select first 10 characters of selected shape</button>
```